sandbox/pages/SandboxCodeBlock.vue

```diff
@@ -42,6 +42,33 @@
           language="json"
         />
       </SandboxSectionComponent>
+      <SandboxSectionComponent
+        class="limited-width syntax-highlighting"
+        title="programmatic highlightedLineNumbers & syntax highlighting"
+      >
+        <KCheckbox
+          v-model="highlightedToggle"
+          label="Highlighted"
+          @change="dirtyCode = true"
+        >
+          Toggle highlighted code
+        </KCheckbox>
+        <br>
+        <KLabel>Showing: {{ highlightedToggle ? 'Code chunk 1 [1-3]' : 'Code chunk 2 [4-6]' }}</KLabel>
+        <KCodeBlock
```

Comment:
suggestion: It might be nice to enable search for this example too so that we can see the whole experience